website/static/js/logTextParser.js

```diff
@@ -285,7 +415,11 @@ var LogPieces = {
         view: function (ctrl, logObject) {
             var source = logObject.attributes.params.source;
             if(paramIsReturned(source, logObject)){
-                return m('span', [source.materialized, ' in ', source.addon]);
+                var sourceMaterialized = source.materialized;
```

Comment:
why not just ` var sourceMaterialized = stripBackslash(source.materialized); `, since stripBacklash already checks if the value ends with a backlash?